### PR TITLE
Updated release version

### PR DESCRIPTION
**Pull request by vatsalgit5118**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 setup(
     name='prancer-basic',
     # also update the version in processor.__init__.py file
-    version='1.0.39',
+    version='1.0.40',
```
Comment (farchide) [inline setup.py]: @vatsalgit5118  change the version to 1.1.0

**Review by farchide — CHANGES_REQUESTED** [review]
change the version to 1.1.0

File: setup.py (hunk 1 above)
Comment (vatsalgit5118) [reply setup.py]: Updated the version to 1.1.0

**Review by farchide — APPROVED** [review]

**Review by ajeybk — APPROVED** [review]
OK, minor version updated

**Pull request merged**